### ISSUE TITLE
Frozen "Audible Login" dialog on first import

### ISSUE THREAD

**Issue by mattalexx** [rmcrackan/Libation#695]
**Describe the bug**
I was following the [very cool] tour and clicked the "Import > Scan Library" menu option. A dialog appeared as an outline only, with a transparent body, and the application froze. I can move the main window but the "Import" dropdown stays frozen in place. I can move the dialog, but I cannot close it.

**To Reproduce**

1. Start First Time User tour.
2. Start first import.
3. See dialog.

**Expected behavior**
I expected a dialog that would ask for my Audible/Amazon password.

**Screenshots**
![](https://i.imgur.com/Hx0uGC4.png)

**Platform**

Arch Linux

```
% uname -a
Linux m 6.4.7-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 27 Jul 2023 22:02:18 +0000 x86_64 GNU/Linux
% kwin_x11 -v
kwin 5.27.6
```

**Log Files**

```
2023-08-02 00:52:08.180 -05:00 [INF] (at AppScaffolding.LibationScaffolding.logStartupState(LibationFileManager.Configuration)) 

 {}2023-08-02 00:52:08.200 -05:00 [ERR] (at LibationFileManager.InteropFactory.get_InteropFunctionsType()) Unable to locate *ConfigApp.dll
 {}2023-08-02 00:52:08.208 -05:00 [INF] (at AppScaffolding.LibationScaffolding.logStartupState(LibationFileManager.Configuration)) Begin. {"AppName":"Libation","Version":"10.6.3.1","ReleaseIdentifier":"LinuxAvalonia","OS":"Linux","InteropFunctionsType":null,"Mode":"Release","LogLevel_Verbose_Enabled":false,"LogLevel_Debug_Enabled":false,"LogLevel_Information_Enabled":true,"LogLevel_Warning_Enabled":true,"LogLevel_Error_Enabled":true,"LogLevel_Fatal_Enabled":true,"BetaOptIn":false,"UseCoverAsFolderIcon":false,"LibationFiles":"/home/matt/Libation","BooksDirectory":"/home/matt/Libation/Books","InProgress":"/tmp/Libation","DownloadsInProgressDirectory":"/tmp/Libation/DownloadsInProgress","DownloadsInProgressFiles":0,"DecryptInProgressDirectory":"/tmp/Libation/DecryptInProgress","DecryptInProgressFiles":0}
 {}2023-08-02 00:52:08.219 -05:00 [WRN] (at AppScaffolding.LibationScaffolding.RunPostMigrationScaffolding(AppScaffolding.Variety, LibationFileManager.Configuration)) WARNING: OSInteropProxy.InteropFunctionsType is null
 {}2023-08-02 00:52:09.778 -05:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts(System.Collections.Generic.IEnumerable`1[[DataLayer.LibraryBook, DataLayer, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]])) Book counts. {"total":0,"booksFullyBackedUp":0,"booksDownloadedOnly":0,"booksNoProgress":0,"booksError":0,"booksUnavailable":0}
 {}2023-08-02 00:52:09.780 -05:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts(System.Collections.Generic.IEnumerable`1[[DataLayer.LibraryBook, DataLayer, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]])) PDF counts. {"total":0,"pdfsDownloaded":0,"pdfsNotDownloaded":0,"pdfsUnavailable":0}
 {}2023-08-02 00:52:09.920 -05:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts(System.Collections.Generic.IEnumerable`1[[DataLayer.LibraryBook, DataLayer, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]])) Book counts. {"total":0,"booksFullyBackedUp":0,"booksDownloadedOnly":0,"booksNoProgress":0,"booksError":0,"booksUnavailable":0}
 {}2023-08-02 00:52:09.921 -05:00 [INF] (at ApplicationServices.LibraryCommands.GetCounts(System.Collections.Generic.IEnumerable`1[[DataLayer.LibraryBook, DataLayer, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]])) PDF counts. {"total":0,"pdfsDownloaded":0,"pdfsNotDownloaded":0,"pdfsUnavailable":0}
 {}2023-08-02 00:52:11.991 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MessageBoxWindow","newValue":"{\n  \"X\": 1268,\n  \"Y\": 505,\n  \"Height\": 110,\n  \"Width\": 316,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:52:12.015 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"AutoScan","newValue":"False"}
 {}2023-08-02 00:52:12.022 -05:00 [INF] (at Dinah.Core.StepRunner.AsyncBaseStep+<RunAsync>d__2.MoveNext()) Begin step ''
 {}2023-08-02 00:52:12.026 -05:00 [INF] (at Dinah.Core.StepRunner.AsyncBaseStep+<RunAsync>d__2.MoveNext()) Begin step 'ShowAccountDialog'
 {}2023-08-02 00:52:13.982 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MessageBoxWindow","newValue":"{\n  \"X\": 1827,\n  \"Y\": 744,\n  \"Height\": 110,\n  \"Width\": 221,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:52:20.131 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MessageBoxWindow","newValue":"{\n  \"X\": 1808,\n  \"Y\": 717,\n  \"Height\": 110,\n  \"Width\": 249,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:52:41.190 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"AccountsDialog","newValue":"{\n  \"X\": 1537,\n  \"Y\": 574,\n  \"Height\": 450,\n  \"Width\": 800,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:52:41.197 -05:00 [INF] (at Dinah.Core.StepRunner.AsyncBaseStep+<RunAsync>d__2.MoveNext()) End step 'ShowAccountDialog'. Success. Completed in 00:00:29
 {}2023-08-02 00:52:41.200 -05:00 [INF] (at Dinah.Core.StepRunner.AsyncBaseStep+<RunAsync>d__2.MoveNext()) Begin step 'ShowSettingsDialog'
 {}2023-08-02 00:52:42.931 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MessageBoxWindow","newValue":"{\n  \"X\": 1835,\n  \"Y\": 744,\n  \"Height\": 110,\n  \"Width\": 204,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:52:56.205 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MessageBoxWindow","newValue":"{\n  \"X\": 1538,\n  \"Y\": 711,\n  \"Height\": 121,\n  \"Width\": 789,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:53:18.960 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"SettingsDialog","newValue":"{\n  \"X\": 1487,\n  \"Y\": 424,\n  \"Height\": 750,\n  \"Width\": 900,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:53:21.967 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MessageBoxWindow","newValue":"{\n  \"X\": 1617,\n  \"Y\": 759,\n  \"Height\": 110,\n  \"Width\": 648,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:53:39.901 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MessageBoxWindow","newValue":"{\n  \"X\": 1552,\n  \"Y\": 747,\n  \"Height\": 134,\n  \"Width\": 778,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:53:49.333 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MessageBoxWindow","newValue":"{\n  \"X\": 1547,\n  \"Y\": 754,\n  \"Height\": 121,\n  \"Width\": 787,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:54:04.475 -05:00 [INF] (at Dinah.Core.StepRunner.AsyncBaseStep+<RunAsync>d__2.MoveNext()) End step 'ShowSettingsDialog'. Success. Completed in 00:01:23
 {}2023-08-02 00:54:04.477 -05:00 [INF] (at Dinah.Core.StepRunner.AsyncBaseStep+<RunAsync>d__2.MoveNext()) Begin step 'ShowAccountScanning'
 {}2023-08-02 00:54:10.761 -05:00 [INF] (at FileManager.PersistentDictionary.writeFile(System.String, Newtonsoft.Json.Linq.JToken)) Config changed. {"propertyName":"MessageBoxWindow","newValue":"{\n  \"X\": 1566,\n  \"Y\": 744,\n  \"Height\": 110,\n  \"Width\": 742,\n  \"IsMaximized\": false\n}"}
 {}2023-08-02 00:54:14.965 -05:00 [INF] (at AudibleUtilities.ApiExtended+<CreateAsync>d__7.MoveNext()) {"LoginType":"ILoginChoiceEager","Account":"AccountId=a[...]n@m[...]r.m[...]e|AccountName=a[...]n@m[...]r.m[...]s|Locale=us","LocaleName":"us"}
 {}
```

**Comment by Mbucari** [comment]
Thanks for the bug report. This will be fixed soon.

**Comment by rmcrackan** [comment]
Fixed in [version 10.6.4](https://github.com/rmcrackan/Libation/releases/latest)
